### DISCUSSION TITLE
Image dimensions for `appLogoUrl` & `appSeoImageUrl`

### DISCUSSION THREAD

**Discussion by crashzk** [ianlucas/cs2-inventory-simulator#103] [General]
I would like to know if there was an image pattern, size to put in these options, change the patterns.

I ask because I tried to create some images, size (1200x630).

ON the website it looks like this:
![image](https://github.com/user-attachments/assets/8422a652-5852-4cdc-9218-51d80bfc803e)

Even when loading the website, the image is also small.

![image](https://github.com/user-attachments/assets/5dc1bdda-19a3-4065-a621-f74a6b5ea5be)

However, SEO Imagem is already good. Would you have any recommended dimensions for them?

**Comment by ianlucas** [comment]
Make sure there is no empty space in the image, only the logo, like the one we use in Inventory Simulator.

![image](https://github.com/user-attachments/assets/e99840e5-012e-4eab-b8ff-c18733742111)
